packages/redux/src/raffles/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,264 @@
+import * as actionTypes from '../actionTypes';
+import {
+  mockCreateRaffleParticipationsNormalizedPayload,
+  mockFetchRafflesNormalizedPayload,
+  mockRaffleEstimationResponse,
+  mockRaffleParticipationResponse,
+  mockRaffleResponse,
+  participationId,
+  raffleId,
+} from 'tests/__fixtures__/raffles';
+import reducer, {
+  entitiesMapper,
+  getAllRaffles,
+  getEstimations,
+  getParticipationCreations,
+  getParticipations,
+  getRaffles,
+  INITIAL_STATE,
+} from '../reducer';
+import type { RafflesState } from '../types';
+
+let initialState: RafflesState;
+
+describe('raffles reducer', () => {
+  beforeEach(() => {
+    initialState = INITIAL_STATE;
+  });
+
+  describe('resetRafflesState', () => {
+    it('should return the initial state', () => {
+      const state = reducer(initialState, {
+        type: actionTypes.RESET_RAFFLES_STATE,
+      });
+      expect(state).toEqual(initialState);
+    });
+  });
+
+  describe('allRaffles reducer', () => {
+    const expectedError = new Error('fetch all raffles error');
+
+    it(`should handle ${actionTypes.FETCH_RAFFLES_REQUEST} action type`, () => {
+      const state = reducer(initialState, {
+        type: actionTypes.FETCH_RAFFLES_REQUEST,
+      });
+      expect(state.allRaffles.isLoading).toBe(true);
+      expect(state.allRaffles.error).toBeNull();
+    });
+
+    it(`should handle ${actionTypes.FETCH_RAFFLES_SUCCESS} action type`, () => {
+      const state = reducer(initialState, {
```

Comment:
You should test with a state that is more representative of when this action is dispatched. In this case, you should test with a state that was set by the `FETCH_RAFFLES_REQUEST` action. Please review all tests that use the `initialState` to use a more representative state if necessary (I see that the `FETCH_RAFFLES_FAILURE` action also suffers from this).

Reply:
done pls double check